lithium/test_lithium.py

```diff
@@ -510,6 +511,29 @@ def test_char_dd(self):
         self.assertEqual(t.parts, [b"d", b"a", b"t", b"a", b"\n", b"2"])
         self.assertEqual(t.after, b"\nDDEND\npost\n")
 
+    def test_jsstr(self):
+        t = lithium.TestcaseJsStr()
+        with open("a.txt", "wb") as f:
+            f.write(b"pre\n")
+            f.write(b"DDBEGIN\n")
+            f.write(b"data\n")
+            f.write(b"2\n")
+            f.write(b"'\\u{123}\"1\\x32\\023\n'\n")  # a str with some escapes
+            f.write(b'""\n')  # empty string
+            f.write(b'"\\u12345"\n')  # another str with the last escape format
+            f.write(b'data\n')
+            f.write(b'"x"\n')  # last str
```

Comment:
These tests are good, but are we missing strings with capital letters? (just to test the regex works as expected with capital letters too)

What about special characters in strings?